packages/sources/coinmetrics-test/src/config/index.ts

```diff
@@ -0,0 +1,52 @@
+import { PriceEndpointInputParameters } from '@chainlink/external-adapter-framework/adapter'
+
+export const DEFAULT_API_ENDPOINT = 'https://api.coinmetrics.io/v4'
+export const DEFAULT_WS_API_ENDPOINT = 'wss://api.coinmetrics.io/v4'
+
+// Used for quote validation and for key names in the response body
+// When changing, be sure to add/remove keys from response body types, such as "WsAssetMetricsResponse"
+export type VALID_REFERENCE_RATE_QUOTES_TYPE = 'USD' | 'EUR' | 'ETH' | 'BTC'
+export const VALID_REFERENCE_RATE_QUOTES: VALID_REFERENCE_RATE_QUOTES_TYPE[] = [
+  'USD',
+  'EUR',
+  'ETH',
+  'BTC',
+]
+
+export const customSettings = {
+  API_KEY: {
+    description: 'The coinmetrics API key',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+} as const
+
+//TODO @ad0ll: Check with lead if we should change this to InputParameters to add validation() functions
```

Comment:
Adding validation would be cool, it's fine to change to InputParameters because using the PriceEndpoint will check for the type anyways